www/src/components/account/CreateGroup.js

```diff
@@ -0,0 +1,88 @@
+import { useMutation } from '@apollo/client'
+import { Box } from 'grommet'
+import {
+  Button,
+  Modal,
+  ModalActions,
+  ModalHeader, ValidatedInput,
+} from 'pluralsh-design-system'
+import { useCallback, useState } from 'react'
+
+import { appendConnection, updateCache } from '../../utils/graphql'
+import { CREATE_GROUP, GROUPS_Q } from '../accounts/queries'
+import { GqlError } from '../utils/Alert'
+
+export function CreateGroup({ q }) {
+  const [open, setOpen] = useState(false)
+  const [name, setName] = useState('')
+  const [description, setDescription] = useState('')
+  const resetAndClose = useCallback(() => {
+    setName('')
+    setDescription(null)
+    setOpen(false)
+  }, [])
+  const [mutation, { loading, error }] = useMutation(CREATE_GROUP, {
+    variables: { attributes: { name, description } },
+    onCompleted: () => resetAndClose(),
+    update: (cache, { data: { createGroup } }) => updateCache(cache, {
+      query: GROUPS_Q,
+      variables: { q },
+      update: prev => appendConnection(prev, createGroup, 'groups'),
+    }),
+  })
+
+  return (
+    <>
+      <Button
+        secondary
+        onClick={() => setOpen(true)}
+      >
+        Create Group
```

Comment:
In the designs it's not using title case: `Create group`.

Reply:
+1—this is the only change required for VQA approval of this PR. Otherwise, everything is looking fantastic. Nice job, @dogmar.